### PR TITLE
OP-213 add search field for opd records

### PR DESCRIPTION
**Pull request by mwithi**
See OP-213.

### CODE REVIEW

File: doc_user/UserManual.adoc [hunk 1]
```diff
@@ -152,17 +152,26 @@ Queries about ambulatory patient visits can be done using the search criteria av
...
-* *Ward*: Select a specific OPD ward
-* *Disease type*: Select a specific disease type or all disease types
-* *Disease*: select a specific disease, or all diseases, or searching for partial matches by typing some letters in the search field
-* *Date*: “Date From” and “Date To” to select all visits occurring during the specified period
-* *Age*: “Age From” and “Age To” to select patients by age
-* *Sex*: Select patients by sex: All / Male / Female
-* *Type of patient*: Select visits by type of attendance: All / New / Re-Attendance
+* *Search by code* (work only one per time)
+** *Code*: Search *ONE* OPD visit by its code, that is unique, hence only one result is expected
+** *OPD No.*: Search OPD visits by their Progressive in Year, that starts every year from one, hence one or more results are expected
```
Comment (mwithi) [inline doc_user/UserManual.adoc]:
```suggestion
** *OPD No.*: Search OPD visits by their Progressive in Year, that starts every year from 1, hence one or more results are expected
```

File: doc_user/UserManual.adoc [hunk 2]
```diff
@@ -152,17 +152,26 @@ Queries about ambulatory patient visits can be done using the search criteria av
...
 
-* *Ward*: Select a specific OPD ward
-* *Disease type*: Select a specific disease type or all disease types
-* *Disease*: select a specific disease, or all diseases, or searching for partial matches by typing some letters in the search field
-* *Date*: “Date From” and “Date To” to select all visits occurring during the specified period
-* *Age*: “Age From” and “Age To” to select patients by age
-* *Sex*: Select patients by sex: All / Male / Female
-* *Type of patient*: Select visits by type of attendance: All / New / Re-Attendance
+* *Search by code* (work only one per time)
+** *Code*: Search *ONE* OPD visit by its code, that is unique, hence only one result is expected
```
Comment (mwithi) [inline doc_user/UserManual.adoc]:
```suggestion
** *Code*: Search an OPD visit by its code, that is unique, hence only *ONE* result is expected
```

File: doc_user/UserManual.adoc [hunk 3]
```diff
@@ -152,17 +152,26 @@ Queries about ambulatory patient visits can be done using the search criteria av
...
 The following fields are available to select or narrow the resulting matches:
 
-* *Ward*: Select a specific OPD ward
-* *Disease type*: Select a specific disease type or all disease types
-* *Disease*: select a specific disease, or all diseases, or searching for partial matches by typing some letters in the search field
-* *Date*: “Date From” and “Date To” to select all visits occurring during the specified period
-* *Age*: “Age From” and “Age To” to select patients by age
-* *Sex*: Select patients by sex: All / Male / Female
-* *Type of patient*: Select visits by type of attendance: All / New / Re-Attendance
+* *Search by code* (work only one per time)
```
Comment (dbmalkovsky) [inline doc_user/UserManual.adoc]:
```suggestion
* *Search by code* (only one can be used at a time)
```

File: doc_user/UserManual.adoc [hunk 4]
```diff
@@ -152,17 +152,26 @@ Queries about ambulatory patient visits can be done using the search criteria av
...
+** *Code*: Search an OPD visit by its code, that is unique, hence only *ONE* result is expected
+** *OPD No.*: Search OPD visits by their Progressive in Year, that starts every year from 1, hence one or more results are expected
+** *Pat. Id*: Search OPD visits by Patient ID, hence one or more results are expected
+
+NOTE: to apply the above *Exclusive Filters* one has to just type a __number__ and press __ENTER__.
+
+* *Other filters* (they are all applied when pressing [.underline]##S##earch)
+** *Ward*: Select a specific OPD ward
+** *Disease type*: Select a specific disease type or all disease types
+** *Disease*: select a specific disease, or all diseases, or searching for partial matches by typing some letters in the search field
```
Comment (dbmalkovsky) [inline doc_user/UserManual.adoc]:
```suggestion
** *Disease*: Select a specific disease, or all diseases, or search using partial matches by typing some letters in the search field
```

**Review by dbmalkovsky — APPROVED** [review]
LGTM with a couple of small changes

**Comment by mwithi** [comment]
> LGTM with a couple of small changes

Done, thanks a lot!

**Pull request merged**